tests/routing/test_path_resolution.py

```diff
@@ -20,22 +20,35 @@ def root_delete_handler() -> None:
 
 
 @pytest.mark.parametrize(
-    "request_path, router_path",
+    "request_path, router_path, status_code",
     [
-        ["/path/1/2/sub/c892496f-b1fd-4b91-bdb8-b46f92df1716", "/path/{first:int}/{second:str}/sub/{third:uuid}"],
-        ["/path/1/2/sub/2535a9cb-6554-4d85-bb3b-ad38362f63c7/", "/path/{first:int}/{second:str}/sub/{third:uuid}/"],
-        ["/", "/"],
-        ["", ""],
+        (
+            "/path/1/2/sub/c892496f-b1fd-4b91-bdb8-b46f92df1716",
+            "/path/{first:int}/{second:str}/sub/{third:uuid}",
+            int(HTTP_200_OK),
+        ),
+        (
+            "/path/1/2/sub/2535a9cb-6554-4d85-bb3b-ad38362f63c7/",
+            "/path/{first:int}/{second:str}/sub/{third:uuid}/",
+            int(HTTP_200_OK),
+        ),
+        ("/", "/", int(HTTP_200_OK)),
+        ("", "", int(HTTP_200_OK)),
+        (
+            "/a/b/c/d/path/1/2/sub/2535a9cb-6554-4d85-bb3b-ad38362f63c7/",
```

Comment:
```suggestion
            "/a/b/c/d/path/1/2/sub/d4aca431-2e02-4818-824b-a2ddc6a64e9c/",
```

This should be a random component as well I think. Currently this is copied from the param above.